src/crimson/osd/pg.h

```diff
@@ -435,8 +435,46 @@ class PG : public boost::intrusive_ref_counter<
   void handle_advance_map(cached_map_t next_map, PeeringCtx &rctx);
   void handle_activate_map(PeeringCtx &rctx);
   void handle_initialize(PeeringCtx &rctx);
-  seastar::future<> handle_op(crimson::net::Connection* conn,
-			      Ref<MOSDOp> m);
+
+  static std::pair<hobject_t, RWState::State> get_oid_and_lock(
+    const MOSDOp &m,
+    const OpInfo &op_info);
+  static std::optional<hobject_t> resolve_oid(
+    const SnapSet &snapset,
+    const hobject_t &oid);
+
+  using load_obc_ertr = crimson::errorator<
```

Comment:
nit, could you rename it to `load_obc_result` ?

Reply:
Why?  It's not the result, it's the type of the errorator.  I'm trying to introduce _ertr as a shorter convention for _errorator.